python/adelphi/adelphi/nb.py

```diff
@@ -172,7 +181,7 @@ def __build_schema(self):
         """Really more of a config than a schema, but we'll allow it"""
         root = {}
 
-        root["scenarios"] = {"TEMPLATE(scenarioname,default)":[self.__get_rampup_scenario(), self.__get_main_scenario()]}
+        root["scenarios"] = {self.scenario_name or TEMPLATEVAR_SCENARIO_NAME:[self.__get_rampup_scenario(), self.__get_main_scenario()]}
```

Comment:
Note that in all three cases the template var is the default case.  If the user doesn't explicitly specify a val via the CLI args we use the template var instead.